keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/NetworkSamplesGenerator.java

```diff
@@ -36,7 +28,39 @@ public NetworkSamplesGenerator(SamplingAlgorithm algorithm, Supplier<ProgressBar
         this.progressBarSupplier = progressBarSupplier;
     }
 
+    public int dropCount() {
```

Comment:
`getDropCount`?

Reply:
This came from from delomboking. If it's getDropCount then it should be setDropCount, which I didn't like and isn't backwards compatible.